addons/koji/common/src/main/java/org/commonjava/indy/koji/inject/KojijiProvider.java

```diff
@@ -143,6 +143,6 @@ public void stop()
     @Override
     public int getShutdownPriority()
     {
-        return 1;
```

Comment:
Too low shutdown priority cause problem. The bean is inactivated by Weld before this stop action is called.